tests/loggers/test_metrics_agg.py

```diff
@@ -0,0 +1,77 @@
+import operator
```

Comment:
not sure if we need these test since it already tested via doctest

Reply:
Yes, you are right. All stuff is tested in docstrings. I removed the tests file.